tests/test_utils.py

```diff
@@ -509,6 +509,50 @@ def test_expand_file_path(tmpdir, index, root, expected):
             pd.DataFrame([0, 1], columns=["name"]),
             "-7179254265801896228",
         ),
+        pytest.param(
+            pd.DataFrame(
+                [0, 1, 2],
+                pd.Index([0, 1, 2], dtype="Int64"),
+            ),
+            "5440931770055407318",
+            marks=pytest.mark.skipif(
+                pd.__version__ >= "2.2.0",
+                reason="Changed in pandas 2.2.0",
+            ),
+        ),
+        pytest.param(
+            pd.DataFrame(
+                [0, 1, 2],
+                pd.Index([0, 1, 2], dtype="Int64"),
+            ),
+            "-5491649331962632325",
+            marks=pytest.mark.skipif(
```

Comment:
We avoid using `audformat.utils.hash()` when calculating the hash for the table PARQUET files, but use a custom method for it to ensure the hash stays always the same (and have also tests for it):

https://github.com/audeering/audformat/blob/642984d700d974111db4840f778a2aba0564b535/audformat/core/table.py#L1847-L1878

The public `audformat.utils.hash()` is used inside `audinterface` for caching of results. But there we use [only the index to estimate the hash](https://github.com/audeering/audinterface/blob/9b6b1d891bb596522e7122edfa7b9a747bbe37d8/audinterface/core/process.py#L601-L604), which means `audinterface` is not affected by the reported change in hash value here. The same is true for `auglib`.

It might be that some users used the hashing for custom caching of dataframes. Those would be affected, but there is nothing we can do about it.

Reply:
> Is the worst consequence that when loading a database using different pandas versions cached parquet files will be downloaded over and over again when two users will use two different pandas versions?

We have ensured that the hash will not change for PARQUET files, but let's assume that this is not true and we have changing hash values. This would not have any consequences when loading the database, as the hash is never verified during that stage. The only time the hash is checked is inside `audb.publish()` as the hash is used in order to estimate which files of the database have changed compared to the previous version. This means the worst consequences would be that identical table files (assuming different hashed) are published again in a new database version, instead of linking to an older version of the same table.

Reply:
>>  This would not have any consequences when loading the database, as the hash is never verified during that stage. The only time the hash is checked is inside audb.publish() as the hash is used in order to estimate which files of the database have changed compared to the previous version. This means the worst consequences would be that identical table files (assuming different hashes) are published again in a new database version, instead of linking to an older version of the same table.

Great, thanks for detailing this. This is the essential bit of information: it is not checked when loading a database, and only affects publishing, and becomes only relevant on every new version of the database. As this is ocurring only infrequently this unproblematic. 

I can resolve this conoversation now as well.